### PR TITLE
bump crystal to 0.20.0

### PR DESCRIPTION
**Pull request by rdp**
also required a kemal bump unfortunately, but I did test it.  I doubt 0.20.0 has any significant performance increase, but just to bring it up to the latest...

### CODE REVIEW

File: frameworks/Crystal/kemal/shard.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ dependencies:
     version: "0.2.3"
   kemal:
     github: "sdogruyol/kemal"
```
Comment (sdogruyol) [inline frameworks/Crystal/kemal/shard.yml]: It's `kemalcr/kemal` from now on

**Comment by rdp** [comment]
OK hopefully got it :)

**Comment by knewmanTE** [comment]
@rdp could you update your repo with the latest from master? We recently merged in a fix that should get Travis working again so we can verify your test.

File: frameworks/Crystal/kemal/shard.lock [hunk 2]
```diff
@@ -1,17 +1,17 @@
 version: 1.0
 shards:
-  html_builder:
-    github: crystal-lang/html_builder
-    version: 0.2.1
-
   kemal:
     github: sdogruyol/kemal
```
Comment (nbrady-techempower) [inline frameworks/Crystal/kemal/shard.lock]: `github: kemalcr/kemal` as well?

**Comment by nbrady-techempower** [comment]
ping @rdp Going to close this for now. If you want to make the next round, which we are starting preview runs for just after the new year, please make the necessary changes and open a new pull request targeting the master branch. Thanks for your continued contributions! :)

**Comment by rdp** [comment]
OK yeah I actually don't know if those would have resulted in any speed improvements anyway, plus crystal is past that version, and kemal will need a future update anyway, so I guess consider this outdated :)

**Pull request closed without merging**